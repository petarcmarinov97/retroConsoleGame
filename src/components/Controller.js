import React, { Component } from 'react';
import "../styles/Controller.css";
import ArrowButtons from './ArrowButtons';
import FunctionalButtons from './FunctionalButtons';

export default class Controller extends Component {

    render() {
        return <div className="controller">
            <>
                <FunctionalButtons />
                <ArrowButtons />
            </>
        </div>;
    }
}
