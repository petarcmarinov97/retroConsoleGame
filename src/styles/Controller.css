div.controller{
    margin-top: 30px;
    width: 410px;
    height: 205px;
    display: flex;
    flex-direction: row;
    align-items: center;
    align-content: center;
    justify-content: space-between;
}