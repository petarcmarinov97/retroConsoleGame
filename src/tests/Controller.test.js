import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import Controller from '../components/Controller';

describe('Controller tests', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Controller/>);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})