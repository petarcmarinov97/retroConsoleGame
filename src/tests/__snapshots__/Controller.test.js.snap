// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controller tests Snapshots should render correctly 1`] = `
<div
  className="controller"
>
  <FunctionalButtons />
  <ArrowButtons />
</div>
`;
